test/fixture/native-app2/package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "private": true,
+  "workspaces": [
```

Comment:
Name the folder `test/fixture/workspace-test` or something of that nature. `native-app2` isn't particularly descriptive of what it's for.